src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Filter } from '../../types/Filter';
+import classNames from 'classnames';
+
+type Props = {
+  status: Filter;
+  onSorted: (sortedField: Filter) => void;
```

Comment:
```suggestion
  onFilterChange: (sortedField: Filter) => void;
```
It's not about sorting 😄